src/test/java/pages/InputPage.java

```diff
@@ -0,0 +1,56 @@
+package pages;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class InputPage extends BasePage {
+    WebElement inputField;
+
+    private String actualResult;
+    public static final By INPUT_FIELD = By.cssSelector("input[type='number']");
+    private String endpoint = "/inputs";
+
+    public InputPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String getActualResult() {
+        return actualResult = inputField.getAttribute("value");
+    }
+
+    @Override
+    public void isPageOpened() {
+        try {
+            wait.until(ExpectedConditions.visibilityOfElementLocated(INPUT_FIELD));
+        } catch (TimeoutException e) {
+            Assert.fail("Input page is not opened");
+        }
+    }
+
+    @Override
+    public void openPage() {
+        driver.get(URL + endpoint);
+    }
+
+    public void input(String text) {
+        inputField = driver.findElement(INPUT_FIELD);
+        inputField.sendKeys(text);
+    }
+
+    public void input(Keys keys) {
```

Comment:
Why? typeTextToInputField for the method with Keys parameters?
I have 2 method with same names "input" but different parameters -  text and keys
Maybe is better off to rename like typeTextToInputField (String text) and tapArrow(Keys keys)? or smth like that

Reply:
Yes, main idea here, try to name methods in a way what they're exactly doing.
typeTextToInputField (String text) - yes
tapArrow(Keys keys) - perhaps better to use smth like: tapKeysToInputField or typeKeysToInputField because it's possible to send not only arrows there.